enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1220 @@
+   in git. The MaintenanceSchedule strategy is chosen. The policy permits control-plane and worker-node updates only after
+   19:00 Eastern US.
+1. During the working day, users may contribute and merge changes to MachineConfigs or even the `desiredUpdate` of the
+   ClusterVersion. These resources will be updated in a timeline manner via GitOps.
+1. Despite the resource changes, neither the CVO nor MCO will begin to initiate the material changes on the cluster.
+1. Privileged users who may be curious as to the discrepancy between git and the cluster state can use `oc get -o=yaml/describe`
+   on the resources. They observe that changes are pending and the time at which changes will be initiated.
+1. At 19:00 Eastern, the pending changes begin to be initiated. This rollout abides by documented OpenShift constraints
+   such as the MachineConfigPool `maxUnavailable` setting.
+   
+#### On-prem Standalone Manual Strategy Scenario
+1. A small, business critical cluster is being run on-prem.
+1. There are no reoccurring windows of time when the cluster lifecycle administrator can tolerate downtime.
+   Instead, updates are negotiated and planned far in advance. 
+1. The cluster workloads are not HA and unplanned drains are considered a business risk.
+1. To prevent surprises, the cluster lifecycle administrator sets the Manual strategy on the worker MCP.
+1. Given the sensitivity of the operation, the lifecycle administrator wants to manually drain and reboot
+   nodes to accomplish the update.
+1. The cluster lifecycle administrator sends a company-wide notice about the period during which service may be disrupted.
+1. The user determines the most recent rendered worker configuration. They configure the `manual` change
+   management policy to use that exact configuration as the `desiredConfig`. 
+1. The MCO is thus being asked to ignite any new node or rebooted node with the desired configuration, but it
+   is **not** being permitted to apply that configuration to existing nodes because it is change management, in effect,
+   is paused indefinitely by the manual strategy.
+1. The MCO metric for the MCP indicating the number of seconds remaining until changes can be initiated is `-1` - indicating
+   that there is presently no time in the future where it will initiate material changes. The operations team
+   has an alert configured if this value `!= -1`.
+1. The MCO metric for the MCP indicating that changes are pending is set because not all nodes are running
+   the most recently rendered configuration. This is irrespective of the `desiredConfig` in the `manual` 
+   policy. Abstractly, it means, if change management were disabled, whether changes be initiated.
+1. The cluster lifecycle administrator manually drains and reboots nodes in the cluster. As they come back online,
+   the MachineConfigServer offers them the desiredConfig requested by the manual policy.

```

Comment:
Yeah, this would just be the administrator annotates (labels?) each node with the new desiredConfig. MCO and MCD handle the rest from there. Administrator is responsible for annotating nodes in order they want and rate limiting that labeling however they want.

Reply:
The description here more matches with what hosted control planes are doing (except that HCP would spin up an entirely new node)